src/shared-with-pluot-core/Environment.js

```diff
@@ -0,0 +1,3 @@
+export function isReactNative() {
```

Comment:
I figure this file could also be used as a home for some other environment checks that are shared between `pluot-core` and `daily-js`, such as browser checks (which are currently duplicated in not-quite-the-same way; see `supportedBrowser()`)